apps/worker/src/app/workflow/services/standard.worker.spec.ts

```diff
@@ -213,7 +223,7 @@ describe('Standard Worker', () => {
       delay: false,
     });
 
-    const jobs = await jobRepository.find({ _environmentId, _organizationId });
+    const jobs = await jobRepository.find({ _environmentId, _organizationId, _notificationId });
```

Comment:
This should have been here already, without it the tests in this suite were interfering with each other.